stdlib/public/core/StringBridge.swift

```diff
@@ -253,8 +253,8 @@ public final class _NSContiguousString : _SwiftNativeNSString {
   }
 
   @objc public // @testable
-  func _fastCStringContents() -> UnsafeMutablePointer<Int8>? {
-    guard _core.elementWidth == 1 else {
+  func _fastCStringContents(_ nullReqd : Bool) -> UnsafeMutablePointer<Int8>? {
```

Comment:
Why not spell the parameter name without abbreviations?  Also, please drop the space before the colon.